docker/stack.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+# Script useful for manage stack saps services life-cycle.
+# See usage for commands & options.
+
+set -o errexit;
+
+readonly ARCHIVER_REPO=wesleymonte/archiver
+readonly CATALOG_REPO=wesleymonte/catalog
+readonly DISPATCHER_REPO=wesleymonte/dispatcher
+readonly SCHEDULER_REPO=wesleymonte/scheduler
+
+readonly ARCHIVER_CONTAINER=saps-archiver
+readonly CATALOG_CONTAINER=saps-catalog
+readonly DISPATCHER_CONTAINER=saps-dispatcher
+readonly SCHEDULER_CONTAINER=saps-scheduler
+
+readonly CATALOG_USER=admin
+readonly CATALOG_PASSWORD=admin
+readonly CATALOG_DB=saps
```

Comment:
same issue as above. You should have an env-composer script that fulfill this fields by reading the sensitive data from a conf file.

Reply:
not sure

Reply:
Believe that it is unnecessary, this script is aimed at the development environment. It is easier to just change the value of the variables.

Reply:
Believe that it is unnecessary, this script is aimed at the development environment. It is easier to just change the value of the variables.